src/main/java/uk/gov/hmcts/reform/sscs/domain/wrapper/FinalDecision.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.sscs.domain.wrapper;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.Objects;
+
+public class FinalDecision {
```

Comment:
Think this can be removed

Reply:
OnlineHearing data object is used by both MYA and COR, can't removed it unless if we modify the frontend